src/Compilers/Core/MSBuildTaskTests/CscTests.cs

```diff
@@ -349,6 +350,14 @@ public void NullableReferenceTypes_False()
             csc.NullableReferenceTypes = false;
             Assert.Equal("/nullable- /out:test.exe test.cs", csc.GenerateResponseFileContents());
         }
+        
+        [WorkItem(29252, "https://github.com/dotnet/roslyn/issues/29252")]
+        public void SdkPath()
+        {
+            var csc = new Csc();
+            csc.SdkPath = @"path\to\sdk";
+            Assert.Equal(@"/sdkpath:path\to\sdk", csc.GenerateResponseFileContents());
+        }
```

Comment:
What are we doing for `sdkpath-` in the build task?

Reply:
Using the `debug` switch as a precedent:

```
-debug[+|-]                   Emit debugging information
-debug:{full|pdbonly|portable|embedded}
                              Specify debugging type ('full' is default,
                              'portable' is a cross-platform format,
                              'embedded' is a cross-platform format embedded into
                              the target .dll or .exe)
```

Relevant snippet from Csc.cs
```cs
// These two -- EmitDebugInformation and DebugType -- must go together, with DebugType 
// getting set last, because it is more specific.
CheckHostObjectSupport(param = nameof(EmitDebugInformation), cscHostObject.SetEmitDebugInformation(EmitDebugInformation));
CheckHostObjectSupport(param = nameof(DebugType), cscHostObject.SetDebugType(DebugType));
```

Snippet from Microsoft.CSharp.Core.targets
```xml
<Csc
     ...
     DebugType="$(DebugType)"
     EmitDebugInformation="$(DebugSymbols)"
     ... />
```

I think we will have to declare another property in the targets file and in Csc/Vbc. Which is kind of unfortunate.